SU-HKKU/cardinal_service_profile
# CSD-442 Added conditional check to add "," and "in"

# READY FOR REVIEW

# Summary
- CSD-442 Added conditional check to add , and in only if the neighboring fields exist

# Needed By (Date)
- Before our meeting this afternoon if possible? :)

# Urgency
- medium

# Steps to Test

1. Pull this pr
2. Clear cache
3. View opportunity page. The comma and "in" should only appear if there are both Dimensions and Type or Commitment and Service tags. 


# Associated Issues and/or People
- [CSD-442](https://stanfordits.atlassian.net/browse/CSD-442)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
